emoji/src/main/java/com/vanniktech/emoji/EmojiSpan.java

```diff
@@ -30,4 +31,9 @@
 
     return drawable;
   }
+
+  @Override public int getSize(final Paint paint, final CharSequence text, final int start,
```

Comment:
Well that's the signature of the [base method](https://developer.android.com/reference/android/text/style/ReplacementSpan.html#getSize(android.graphics.Paint,java.lang.CharSequence,int,int,android.graphics.Paint.FontMetricsInt)), so we have to specify all of these parameters.